community-operators/knative-serving-operator/knative-serving-operator.v0.5.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,209 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
```

Comment:
Please reduce the clusterPermissions for this operator to roles that are needed by the operator.

Reply:
I'm afraid `cluster-admin` is the minimum required due to its use of `aggregationRules`. 
/cc @pmorie

Reply:
@dmesser This is what we discussed over email ^^